doomgrader.sh

```diff
@@ -39,5 +66,9 @@ sed -i 's/dotnet/mono/' depotdownloader
 ./depotdownloader -app 782330 -depot 782336 -manifest 4248922069342282231 -username "$STEAM_USERNAME" -password "$STEAM_PASSWORD" -remember-password -dir "$DOWNLOAD_PATH"
 ./depotdownloader -app 782330 -depot 782339 -manifest 8937962102049582968 -username "$STEAM_USERNAME" -password "$STEAM_PASSWORD" -remember-password -dir "$DOWNLOAD_PATH"
 
-# copy game files to steam dir
-\cp $DOWNLOAD_PATH/* $STEAM_PATH/DOOMEternal/ -rf
+# adds space between depot downloader output and DOOMGRADER_COPY prompt
```

Comment:
I don't think we need to worry about formatting

Reply:
If we keep the -v in the cp command, I think it is important to put the newline in there for users, as it is hard to read all of the prompts when there is a lot of cp output. This echo "" helps to separate this question from the output

Reply:
Ok sure

Reply:
Awesome, will do the commit